src/client/Game.jsx

```diff
@@ -157,6 +185,21 @@ export default class Game extends React.Component {
     }))
   }
 
+  rematch (action) {
+    if(action === 'request'){
+      this.setState({
+        game: {  ...this.state.game, status: common.gameStatus.STATUS_AWAIT_REMATCH_RESPONSE }
```

Comment:
two spaces in `{  ...this.state.game`, use only one

Reply:
Fixed.